apis/core/v1alpha1/types_deployitem.go

```diff
@@ -83,6 +90,10 @@ type DeployItemStatus struct {
 	// LastError describes the last error that occurred.
 	LastError *Error `json:"lastError,omitempty"`
 
+	// LastChangeReconcileTime indicates when the reconciliation of the last change to the deploy item has started
+	// +optional
+	LastChangeReconcileTime *metav1.Time `json:"lastChangeReconcileTime,omitempty"`
```

Comment:
I think that the naming is a little bit long.
I think `LastReconcileTime` also describes what this field indicates. wdyt?

Reply:
The problem that I have with this is that we also use the term `reconcile` for each call to the deployers `Reconcile` method. We are 'reconciling' a deploy item a lot more than it actually changes, but this value is meant to contain the timestamp of the time when we started _reconciling the latest change to the deploy item_. That's why in my opinion `lastReconcileTime` is somewhat imprecise and I don't really like it. 
I get your point though, `lastChangeReconcileTime` is not really nice either. 

Maybe we can find some other term for this ... how about these:
- LastChangeReconciled (basically dropped the 'time' suffix)
- LastChanged (also imprecise, as it refers to the time the deployer has picked up the change and not the change itself ...)
- LastChangePickup